lib/Doctrine/DBAL/Query/QueryBuilder.php

```diff
@@ -409,7 +409,7 @@ public function getMaxResults()
      * 'groupBy', 'having' and 'orderBy'.
      *
      * @param string  $sqlPartName
-     * @param string  $sqlPart
+     * @param mixed  $sqlPart
```

Comment:
In some part of the code for example in line 530 is passed as an array not as a string right?